endroid/asset/1.0/manifest.json

```diff
@@ -1,8 +0,0 @@
-{
-    "bundles": {
-        "Endroid\\Asset\\Bundle\\AssetBundle\\EndroidAssetBundle": ["all"]
-    },
```

Comment:
At least this section is required to register the bundle automatically.

Reply:
Hi @yceruto endroid/asset does not exist anymore because it provided functionality that is now provided by Symfony itself. That means the manifest can also be removed completely. Otherwise it would only refer to a non-existent package.

Reply:
Means the recipe can be completely eliminated?

Reply:
Yes, correctly, no need for it anymore. Thank you.